docs/Analyzer Configuration.md

```diff
@@ -741,3 +741,15 @@ Configurable Rules: [CA1711](https://docs.microsoft.com/visualstudio/code-qualit
 Option Values: List (separated by '|') of allowed suffixes
 
 Example: `dotnet_code_quality.CA1711.allowed_suffixes = Flag|Flags`
+
+### Enable platform compatibility analyzer for TFMs <= net5.0
+
+Option Name: `enable_platform_analyzer`
+
+Configurable Rules: [CA14016](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1416)
```

Comment:
```suggestion
Configurable Rules: [CA1416](https://docs.microsoft.com/visualstudio/code-quality/ca1416)
```

Reply:
Good catch, @mavasani JFYI the path copied from the above rules, all are having `en-us`

Reply:
Thanks, I'll create a separate PR to bulk remove `/en-us` from this page.